doc/3/controllers/collection/validate-specifications/index.md

```diff
@@ -0,0 +1,60 @@
+---
+code: true
+type: page
+title: validateSpecifications
+description: Validate specifications format
+---
+
+# validateSpecifications
+
+The validateSpecifications method checks if a validation specification is well formatted. It does not store or modify the existing specification.
+
+When the validation specification is not formatted correctly, a detailed error message is returned to help you to debug.
+
+<br/>
+
+```java
+  public CompletableFuture<ConcurrentHashMap<String, Object>> validateSpecifications(
+      final String index,
+      final String collection,
+      final ConcurrentHashMap<String, Object> specifications)
+```
+
+<br/>
+
+| Arguments        | Type                                         | Description                |
+| ---------------- | -------------------------------------------- | -------------------------- |
+| `index`          | <pre>String</pre>                            | Index name                 |
+| `collection`     | <pre>String</pre>                            | Collection name            |
+| `specifications` | <pre>ConcurrentHashMap<String, Object></pre> | Specifications to validate |
+
+### specifications
+
+A `ConcurrentHashMap<String, Object>` representing the specifications.
+
+This object must follow the [Specification Structure](/core/2/guides/cookbooks/datavalidation):
+
+```java
```

Comment:
neither a java example, nor a json one

Reply:
I've let those examples because it looks like this when you print it out
But I see how it's note accurate, i will remove it